wp-content/themes/access/src/js/modules/screener.js

```diff
@@ -454,6 +454,10 @@ class Screener {
       this._renderRecap();
     }
 
+    let stepTitle = $(section).find('[data-js="step-title"]').text();
+    let baseTitle = 'Eligibility Screener – ACCESS NYC - ';
```

Comment:
This is great work. One fix we'll want to make is using the proper canonical title for the page. In our case it should read be `Step 1 of 10 - Eligibility Screener - ACCESS NYC`. Also, instead of storing this string in the script, we should retrieve it from the DOM on page load. We can try this; store the base title in the `constructor()` method of the screener class;

```
constructor(el) {
  ...
  this._baseTitle = $('title').text();
  ...
}
```

Reply:
@devowhippit would I have to add the datatype for the `_baseTitle` for example:
```
/** @private {string} The component element. */
this._baseTitle = $('title').text();```

Reply:
@theasteve correct.